config/application.yml.default

```diff
@@ -98,6 +98,7 @@ event_disavowal_expiration_hours: 240
 feature_idv_force_gpo_verification_enabled: false
 feature_idv_hybrid_flow_enabled: true
 feature_new_device_alert_aggregation_enabled: true
+feature_select_email_to_share_enabled: true
```

Comment:
What's the plan for enabling this feature? Do we have follow-on tickets?

Reply:
I will add a follow up for that.

Reply:
https://cm-jira.usa.gov/browse/LG-14236